src/toast/ToastProvider.tsx

```diff
@@ -3,6 +3,7 @@ import ReactDOM from "react-dom";
 import { canUseDOM } from "reakit-utils";
 import { createContext } from "@chakra-ui/utils";
```

Comment:
Extract the utils from chakra - https://github.com/chakra-ui/chakra-ui/blob/develop/packages/utils/src/react-helpers.ts 🤔 to our local utils?

Reply:
What about this @anuraghazra  ?

Reply:
I will extract it out...